src/main/resources/hapi-r4.properties

```diff
@@ -0,0 +1,151 @@
+
+###################################################
+# Supported Resources
+###################################################
+# Enable the following property if you want to customize the
+# list of resources that is supported by the server (i.e. to
+# disable specific resources)
+#supported_resource_types=Patient,Observation,Encounter
+
+###################################################
+# Database Settings
+###################################################
+hibernate.dialect=org.hibernate.dialect.H2Dialect
+hibernate.search.model_mapping=ca.uhn.fhir.jpa.search.LuceneSearchMappingFactory
+hibernate.format_sql=false
+hibernate.show_sql=false
+hibernate.hbm2ddl.auto=update
+hibernate.jdbc.batch_size=20
+hibernate.cache.use_query_cache=false
+hibernate.cache.use_second_level_cache=false
+hibernate.cache.use_structured_entries=false
+hibernate.cache.use_minimal_puts=false
+hibernate.search.default.directory_provider=filesystem
+hibernate.search.default.indexBase=target/lucenefiles
+hibernate.search.lucene_version=LUCENE_CURRENT
+tester.config.refuse_to_fetch_third_party_urls=false
+
+##################################################
+# ElasticSearch
+# Note that using ElasticSearch is disabled by
+# default and the server will use Lucene instead.
+##################################################
+elasticsearch.enabled=false
+elasticsearch.rest_url=http://localhost:9200
+elasticsearch.username=SomeUsername
+elasticsearch.password=SomePassword
+elasticsearch.required_index_status=YELLOW
+elasticsearch.schema_management_strategy=CREATE
+# Immediately refresh indexes after every write. This is very bad for
+# performance, but can be helpful for testing.
+elasticsearch.debug.refresh_after_write=false
+elasticsearch.debug.pretty_print_json_log=false
+
+
+##################################################
+# Binary Storage Operations
+##################################################
+binary_storage.enabled=true
+
+##################################################
+# Bulk Data Specification
+##################################################
+bulk.export.enabled=true
+
+##################################################
+# CORS Settings
+##################################################
+cors.enabled=true
+cors.allowCredentials=true
+# Supports multiple, comma separated allowed origin entries
+# cors.allowed_origin=http://localhost:8080,https://localhost:8080,https://fhirtest.uhn.ca
+cors.allow_origin=*
+
+##################################################
+# Allowed Bundle Types for persistence (defaults are: COLLECTION,DOCUMENT,MESSAGE)
+##################################################
+#allowed_bundle_types=COLLECTION,DOCUMENT,MESSAGE,TRANSACTION,TRANSACTIONRESPONSE,BATCH,BATCHRESPONSE,HISTORY,SEARCHSET
+
+##################################################
+# Subscriptions
+##################################################
+
+# Enable REST Hook Subscription Channel
+subscription.resthook.enabled=false
```

Comment:
Do we want this to be true

Reply:
I will be committing these changes in next commit,
This is the first commit for custom property file